WORKSPACE

```diff
@@ -347,13 +346,6 @@ http_archive(
     url = PYTHON_PLUGIN_233_URL,
 )
 
-http_archive(
-    name = "python_2023_1",
-    build_file_content = _PYTHON_CE_BUILD_FILE,
-    sha256 = "825c30d2cbcce405fd18fddf356eb1f425607e9c780f8eff95d21ac23f8d90fd",
-    url = "https://plugins.jetbrains.com/maven/com/jetbrains/plugins/PythonCore/231.8770.65/PythonCore-231.8770.65.zip",
-)
-
```

Comment:
This rule seems redundant so deleting here. I had to delete this otherwise I get errors:
```
Error in fail: generate_repo_config: generate_repo_config: /workdir/WORKSPACE: multiple rules have the name "python_2023_1"
```
Link: https://buildkite.com/bazel/intellij-plugin-aspect/builds/20467